app/src/main/java/com/kelvin/bdaypost/birthday/add/AddBirthdayFragment.kt

```diff
@@ -0,0 +1,159 @@
+package com.kelvin.bdaypost.birthday.add
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import com.kelvin.bdaypost.BuildConfig
+import com.kelvin.bdaypost.birthday.data.model.Birthday
+import com.kelvin.bdaypost.birthday.data.model.Month
+import com.kelvin.bdaypost.databinding.FragmentAddBirthdayBinding
+import com.kelvin.bdaypost.util.TextResUtil.showToast
+import timber.log.Timber
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [AddBirthdayFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class AddBirthdayFragment : Fragment() {
+    private var _addBdayBinding: FragmentAddBirthdayBinding? = null
+    // only use this between onCreateView and onDestroyView
+    private val addBdayBinding
+        get() = _addBdayBinding!!
+
+    private var addBdayVM: AddBirthdayViewModel? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _addBdayBinding = FragmentAddBirthdayBinding.inflate(inflater, container, false)
+        return addBdayBinding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        // create VM
+        addBdayVM = activity?.run {
```

Comment:
Is there a reason for using the activity rather than the fragment as the view model owner?